build.gradle

```diff
@@ -35,7 +35,7 @@ apply plugin: 'io.codearte.nexus-staging'
 
 allprojects {
     group = 'io.minio'
-    version = '6.0.12'
+    version = '6.0.13'
```

Comment:
```suggestion
    version = '6.0.12'
```

Reply:
As per discussion with @balamurugana  we will have two listenBucketNotification method. Old one will be depricated. After some time we can have a major release in which all the deprecated methods will be removed .